src/features/swagger/index.ts

```
@@ -1 +1 @@
-export { default as Swagger } from './components/Swagger/Swagger';
+export { default } from './components/Swagger/Swagger';
```

ESLint

[error] 'default' is restricted from being used as an exported name.

(no-restricted-exports)